### DISCUSSION TITLE
want to remove nautilus but says will break dependency

### DISCUSSION THREAD

**Discussion by efwis** [mylinuxforwork/dotfiles#276] [Q&A] [answered]
I'm wanting to remove nautilus and use nemo for preferential reasons, i have nemo working fine, but when i try to uninstall nautilus it pops up cannot remove nautilus because ml4w-hyprland dependency.

how can I uninstall nautilus so I don't have software packages in my system i don't need/want?

ETA: I'm on EndeavourOS if that matters

**Comment by hambosto** [comment, marked as answer]
just type sudo pacman -Rdd nautilus

**Reply by efwis** [reply]
awesome thank you, guess i need to read up on pacman a little more in the wiki, lol
